Can't resolve 'worker_thread'

It seems the changes done in  https://github.com/pmndrs/use-cannon/blob/5ff2baacd7caff2079b21b42ea9ca7798070a5d9/rollup.config.js altered what was done in https://github.com/pmndrs/use-cannon/commit/37fcb7dd74d2456289c4047fd06bf01a4c77e4d6 to fix this issue (#86), making it return:

This is what I get when trying to `npm run build`:
```
Module not found: Error: Can't resolve 'worker_threads' in `.../node_modules/@react-three/cannon/dist'
```

I am not well-versed enough in React/npm to be able to add much to this, sorry! All I can say is that switching back to `3.7.0` (the version prior to the breaking commit) fixed the issue.

I'm also having the same issue. I'm using `webpack` for bundling and building my app. Here's the code:
```ts
// Core.tsx
import { Physics } from '@react-three/cannon';
```
Error from `webpack`:
```ts
WARNING in ./node_modules/@react-three/cannon/dist/index.js 11:81-113
Module not found: Error: Can't resolve 'worker_threads' in '/home/kyuzu/Project/portfolio/node_modules/@react-three/cannon/dist'
 @ ./components/Core.tsx 4:0-46 40:42-49
 @ ./app/App.tsx 2:24-52
 @ ./app/index.tsx 3:0-15
 @ ./src/app.ts 1:0-22
 @ ./src/index.ts 2:0-15
 ```

--- a/packages/react-three-cannon/rollup.config.js
+++ b/packages/react-three-cannon/rollup.config.js
@@ -25,7 +25,7 @@ export default [
     output: { dir: 'dist', format: 'esm' },
     plugins: [
       resolve({ extensions }),
-      worker({ platform: 'base64', sourcemap: false }),
+      worker({ platform: 'base64', sourcemap: false, targetPlatform: 'browser' }),
       babel(getBabelOptions({ useESModules: true }, '>1%, not dead, not ie 11, not op_mini all')),
     ],
   },
@@ -35,7 +35,7 @@ export default [
     output: { dir: 'dist/debug', format: 'esm' },
     plugins: [
       resolve({ extensions }),
-      worker({ platform: 'base64', sourcemap: true }),
+      worker({ platform: 'base64', sourcemap: true, targetPlatform: 'browser' }),
       babel(getBabelOptions({ useESModules: true }, '>1%, not dead, not ie 11, not op_mini all')),
     ],
   },

